src/components/forms/constants.ts

```diff
@@ -1483,6 +1492,15 @@ export const APPLYING_AS_OPTIONS = [
   { value: 'Other', label: 'Other' }
 ];
 
+export const APPLYING_AS_DATA_COLLECTION_OPTIONS = [
```

Comment:
This const should be removed and should use `APPLYING_AS_OPTIONS` instead

Reply:
Nice, thanks!